app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/cases_list.scala.html

```diff
@@ -24,7 +24,7 @@
     <thead class="govuk-table__head">
         <tr class="govuk-table__row">
             <th class="govuk-table__header" scope="col">Reference</th>
-            <th class="govuk-table__header" scope="col">Description</th>
+            <th class="govuk-table__header" scope="col">Item Name</th>
```

Comment:
I think the classification team prefers `Product Name`

Reply:
This was Joes suggestion not mine